nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw fix api file location

# Description
API removal; Run fetch and set user and techstack states to App.js
## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions

1. Pull down branch. Run it in browser.
2. Check that users display in the Dev List

# Checklist:
